service-front/module/Application/src/Forms/PostOfficeAddress.php

```diff
@@ -6,15 +6,17 @@
 
 use Laminas\Form\Annotation;
 use Laminas\Hydrator\ObjectPropertyHydrator;
+use Laminas\Validator\NotEmpty;
 
 /**
  * @psalm-suppress MissingConstructor
  */
 #[Annotation\Hydrator(ObjectPropertyHydrator::class)]
-class PostOfficeNumericCode
```

Comment:
Switching this from the fad code to the full address, is this going to have any downstream consequences?

Reply:
The attrType in the API was always a string, so it should be fine (and works in testing without any issues)